modules.d/99openssl/module-setup.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+check() {
+    return 255
+}
+
+install() {
+    local ossl_files="${dracutbasedir}/ossl-files"
+
+    local openssl_cnf="$($ossl_files --config)"
```

Comment:
```suggestion
    local openssl_cnf="$($ossl-files --config)"
```

As I understand the Makefiles, you installed this as `ossl-files`. The same applies below.

Reply:
I don't have it in PATH, but I have it in dracut dir, so this is a shell variable instead, difined two lines above this one :). Sorry for the confusion.

Reply:
Ah, you're right. I was confused because I missed the second `$` in front.

Technically, this should be `"$("$ossl_files" --config)"`, then, though. Although it's unlikely that there ever will be a space in `$ossl_files`.